backends/arm/_passes/unsqueeze_scalar_placeholders_pass.py

```diff
@@ -19,23 +20,27 @@ def __init__(self, exported_program):
         self.exported_program = exported_program
         super().__init__()
 
-    def _is_inputs_to_buffers_or_parameters(self, node):
-        return (
-            node.name in self.exported_program.graph_signature.inputs_to_buffers
-            or node.name in self.exported_program.graph_signature.inputs_to_parameters
-        )
-
     def call(self, graph_module: torch.fx.GraphModule):
         for node in graph_module.graph.nodes:
             if node.op != "placeholder":
                 continue
             rank = node.meta["val"].dim()
             if rank == 0:
-                if not self._is_inputs_to_buffers_or_parameters(node):
+                if is_buffer(self.exported_program, node):
```

Comment:
nit make a util `def get_name(self, node)`?

Reply:
Local to this file? Seems a little redundant to me, I suppose that would look something like this

```
def get_name(self, node):
     if is_buffer(self.exported_program, node):
         return self.exported_program.graph_signature.inputs_to_buffers[node.name]
     # Reapeat for param etc.
     ...
```

And this is done 2 times only.

I think it would belong in torch._export.utils like is_buffer etc.